docker-compose.gcb.yaml

```diff
@@ -0,0 +1,30 @@
+---
+version: '3.4'
+services:
+  postgres:
+    build: 
+      context: .github/workflows/postgres
```

Comment:
Err, why are we building our own Postgres image, instead of using one from https://hub.docker.com/_/postgres?

Reply:
Because it needs a custom `docker-entrypoint-initdb.d` directory to allow replications connections. There may be a way to set that up on an existing image but I never manage to successfully do that.

Reply:
Figured out how to do this with volumes. Now we are not building the image.